po/README.md

```diff
@@ -1,22 +1,214 @@
 # Translating Extension Manager
+Translations contributions is always welcome!
```

Comment:
```suggestion
Translations contributions are always welcome!
```

Reply:
Since I'm also not native speaker, I think, it would be good idea to wait for someone who speak English natively?
Should I apply suggestions now or wait?